python/tvm/relay/frontend/tensorflow.py

```diff
@@ -887,7 +887,14 @@ def _impl(inputs, attr, params):
         if 'U' in attr:
             need_cast = True
             inputs[0] = _op.cast(inputs[0], dtype=attr['U'].name)
-
+        # Check if mean and variance are empty
+        # If so, replace them with Mean and Variance Ops
+        # For run-time calculation
+        moving_mean_shape = [int(n) for n in inputs[3].type_annotation.shape]
```

Comment:
It is the time to add `CHECK` of `len(inputs)`

Reply:
Thank you for the detailed review comments! I have corrected the related codes in the newest commit. But for the `add CHECK of len(inputs)` issue, does it mean that to check the number of nodes in `inputs` and to see whether this number is 5? Also, it seems in python there is no `CHECK` method?
Thank you so much for the help!

Reply:
The `CHECK` means `assert`. 

Yes, `CHECK` the number of nodes, so that we could visit `inputs[3]` / `inputs[4]` and so on safely.

Reply:
I see. Thank you so much!